sdk/tools/gen_baseaddress.py

```diff
@@ -351,7 +358,7 @@ def get_target_file(ntdll_path):
         if b'.rossym' in names:
             return 'baseaddress.cmake'
         elif is_x64():
```

Comment:
Now my local sdk/tools/gen_baseaddress.py generatesthe file baseaddress_dwarf.cmake

Reply:
You have to specify -64 as first parameter to the script.

Reply:
Okky, sorry I misread that part of the code. My baseaddress fits like this:
[baseaddress64.cmake.txt](https://github.com/reactos/reactos/files/13196604/baseaddress64.cmake.txt)